src/Suave/Http.fsi

```diff
@@ -331,6 +331,16 @@ module Http =
   /// A server-key is a 256 bit key with high entropy
   type ServerKey = byte []
 
+  /// Utilities to ensure server keys are well-formed
+  [<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+  module ServerKey =
+    
+    /// Ensure that a server key is the proper length
+    val create : byte [] -> ServerKey
```

Comment:
Up until I sat down this morning to wrap this up, it literally was.  :)  I can change it back; my thought was that a `byte []` is unknown, but a `ServerKey` is a verified key.  I know that the alias definition means that there's technically no difference between `byte []` and `ServerKey`.  It also seemed that most of the modules have a `create` function.

Let me know if you want me to change it; it's an easy fix.